src/azul/plugins/repository/tdr/__init__.py

```diff
@@ -204,6 +204,7 @@ def retrieve_rows(entity_type: str, entity_ids: Set[str]):
             non_pk_columns = bundle.data_columns if entity_type.endswith('_file') else bundle.metadata_columns
             columns = ', '.join(non_pk_columns | {pk_column})
             uuid_in_list = ' OR '.join(f'{pk_column} = "{entity_id}"' for entity_id in entity_ids)
+            log.info('Retrieving %s %s entities...', len(entity_ids), entity_type)
```

Comment:
```suggestion
            log.info('Retrieving %i entities of type %r ...', len(entity_ids), entity_type)
```

similarly for the log message at the conclusion of this method.